sdk/schemaregistry/schema-registry/test/public/schemaRegistry.spec.ts

```diff
@@ -64,11 +64,13 @@ describe("SchemaRegistryClient", function () {
   let client: SchemaRegistryClient;
   let schema: SchemaDescription;
 
-  beforeEach(function (this: Context) {
-    ({ client, recorder } = createRecordedClient(this));
+  beforeEach(async function (this: Context) {
+    recorder = new Recorder(this.currentTest);
+    await recorder.start(recorderOptions);
+    client = createRecordedClient(recorder);
     schema = {
       name: "azsdk_js_test",
-      groupName: env.SCHEMA_REGISTRY_GROUP,
+      groupName: getEnvironmentVariable("SCHEMA_REGISTRY_GROUP"),
```

Comment:
This feels like a regression to have to call a function to get an environment variable instead of being able to access it as a property of an object.

Reply:
Should I make env to be `Record<string, string>` then?

Reply:
@witemple-msft you do not have to call the function, you just need to check whether the field you're accessing exists.

Reply:
> Should I make env to be `Record<string, string>` then?

Okay I see now that the reason the function exists is because it _asserts_ that an environment variable is defined, but that you can still get the environment variable from `env`, it just might be undefined. I would not change it to Record<string, string>.

It'd be helpful if the function name somehow conveys that it is not just getting the value of an environment variable, but will throw if it is not defined i.e. `assertEnvironmentVariable`, or `getEnvStrict`, etc.

Reply:
Updated. Calling it `assertEnvironmentVariable` now.

Any objections? @deyaaeldeen @jeremymeng